feature/deps.go

```diff
@@ -58,6 +58,9 @@ var deps = depDesc{
 	lnwire.AnchorsOptional: {
 		lnwire.StaticRemoteKeyOptional: {},
 	},
+	lnwire.ZeroHtlcTxFeeOptional: {
+		lnwire.AnchorsOptional: {},
```

Comment:
we may or may not need to modify this depending on the decisions made in https://github.com/lightningnetwork/lightning-rfc/pull/824

Reply:
if we remove the dependency we can also stop advertising `AnchorsOptional`

Reply:
Dependency removed!